WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/TilesKotlinScreen.kt

```diff
@@ -21,20 +21,33 @@ import androidx.annotation.StringRes
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
 import androidx.wear.compose.material.Chip
 import androidx.wear.compose.material.ListHeader
 import androidx.wear.compose.material.ScalingLazyColumn
 import androidx.wear.compose.material.Text
 import androidx.wear.compose.material.rememberScalingLazyListState
 import androidx.wear.tiles.TileService
+import com.google.android.horologist.compose.navscaffold.ExperimentalHorologistComposeLayoutApi
+import com.google.android.horologist.compose.navscaffold.scrollableColumn
+import com.google.android.horologist.compose.pager.FocusOnResume
 
+@OptIn(ExperimentalHorologistComposeLayoutApi::class)
 @Composable
-internal fun SampleTilesList(context: Context, sampleTiles: Map<Int, Class<out TileService>>) {
+internal fun TilesKotlinScreen(
+    modifier: Modifier = Modifier,
+    context: Context,
+    sampleTiles: Map<Int, Class<out TileService>>,
+    focusRequester: FocusRequester = remember { FocusRequester() }
+) {
     TilesKotlinTheme {
         val listState = rememberScalingLazyListState()
         ScalingLazyColumn(
-            modifier = Modifier.fillMaxSize(),
+            modifier = modifier
+                .fillMaxSize()
```

Comment:
I'd probably assume the caller is deciding this part, possibly the parent is setting `fillMaxSize(0.9)` and you are overriding.

Reply:
Is this the case for composables that are intended to be the root screen, or is "root screen" an assumption that doesn't always hold?